app/src/main/java/unipd/se18/ocrcamera/BrightnessRecognition.java

```diff
@@ -0,0 +1,93 @@
+package unipd.se18.ocrcamera;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Color;
+import android.util.Log;
+import android.widget.Toast;
+import android.os.Handler;
+
+/**
+ * This class is used to calculate the brightness of an image in Bitmap format
+ * @author Pietro Balzan
+ */
+public class BrightnessRecognition {
+    Handler handler;             // handler for communication with the ui thread
+    private Context appContext;  //context of the activity that instances the object of this class
+    private final Bitmap image;  // the bitmap containing the image to analyze
+    private static final String TAG = "BrightnessRecognition";
+
+
+    /**
+     * Constructor of a BrightnessRecognition object containing a Bitmap
+     * @param bmp the bitmap of the image to calculate the brightness of
+     * @param cntxt context of the activity that instances an object of this class
+     * @param hndlr handler for communication with ui thread
+     * @modify image with the new bitmap
+     * @author Pietro Balzan
+     */
+
+    public BrightnessRecognition(Bitmap bmp, Context cntxt, Handler hndlr){
```

Comment:
Could have sense if this class contains only static methods without a constructor?

Reply:
It makes sense